lib/screens/generic_item_selector_screen.dart

```diff
@@ -43,34 +39,27 @@ class _GenericItemSelectorScreenState extends State<GenericItemSelectorScreen> {
   void initState() {
     _controller = ScrollController();
     _controller.addListener(_scrollListener); //the listener for up and down.
+
+    // todo(TurnipXenon)
+    clothsNotifier.value = new List<clothing>.from(clothings);
+
     super.initState();
   }
 
-  Widget _buildIcon(int index) {
-    return GestureDetector(
-      onTap: () {
-        setState(() {
-          _selectedIndex = index;
-        });
-      },
-      child: Container(
-        height: 60.0,
-        width: 60.0,
-        decoration: BoxDecoration(
-          color: _selectedIndex == index
-              ? Theme.of(context).accentColor
-              : Color(0xFFE7EBEE),
-          borderRadius: BorderRadius.circular(30.0),
-        ),
-        child: Icon(
-          _icons[index],
-          size: 25.0,
-          color: _selectedIndex == index
-              ? Theme.of(context).primaryColor
-              : Color(0xFFB4C1C4),
-        ),
-      ),
-    );
+  // todo(TurnipXenon): run async
```

Comment:
TODO!

-[ ] Replace with SQL queries; code here is for demo to show the list can update once the tags are changed